kernel/api/src/main/java/org/sakaiproject/content/api/ContentHostingService.java

```diff
@@ -113,6 +113,8 @@ public interface ContentHostingService extends EntityProducer
 
 	/** Name of the event when reading a resource. */
 	public static final String EVENT_RESOURCE_READ = "content.read";
+	
+	public static final String EVENT_RESOURCE_ZIP_DOWNLOAD = "content.zipDownload";
```

Comment:
I think the whole ticket may have a wrong description, I think they want to reflect an event when users compress resources in ZIP, could you confirm @vacercon  ?

Reply:
This should be 

"content.zip.download"

Reply:
Alright. I will do the changes